apps/web/src/pages/quick-start/consts.tsx

```diff
@@ -195,3 +205,17 @@ export const frameworkInstructions: { key: string; value: ISnippetInstructions[]
     ],
   },
 ];
+
+export enum OnBoardingAnalyticsEnum {
+  FRAMEWORK_SETUP_VISIT = '<FRAMEWORK> Setup Page Visit',
```

Comment:
I added those enum params with generic params like <FRAMEWORK> so it would be easier to manage our events (all the events are in one place plus you can easily point to every event from here).
Would love to hear if there are any other opinions on the matter.

Reply:
I would recommend to do it in the following structure:

```
.trach("In app frameworks select", { framework })
```

This way we can easily track all counts, and filter/breakdown if needed

Reply:
Thanks for the suggestion,I didn't have a chance to look into those logs and how they are presented in the end.